utils/docker/base-dev/Dockerfile

```diff
@@ -86,7 +86,7 @@ RUN go install golang.org/x/tools/cmd/goimports@latest \
   && go install github.com/maxbrunsfeld/counterfeiter/v6@latest \
   && go install honnef.co/go/tools/cmd/staticcheck@2023.1.3 \
   && go install github.com/client9/misspell/cmd/misspell@latest \
-  && go install github.com/mikefarah/yq/v3@latest
+  && go get github.com/mikefarah/yq/v4@latest
```

Comment:
I believe this should be `go install ...`

Reply:
I thank you pointing out the error and fixing it.